e2e/user/user-can-cancel-edit-account.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { test, expect } from "@playwright/test";
+import { loginKnownUser } from "../_lib/pom/init/login-known-user";
+import { gotoUserMyAccountPage } from "../_lib/sequences/nav-gotos";
+import { UserMyAccountEditPage } from "../_lib/pom/pages/user-my-account-edit-page";
+
+test("user can login, go to edit page, but cancel, and should not see their details changed", async ({
+  page,
+}) => {
+  // GIVEN
+  const { context, knownUser } = await loginKnownUser({ page });
+  const accountPage = await gotoUserMyAccountPage({ context });
+  const { userName, userEmail, userPhoneNumber, userResidency } = knownUser;
+  await expect(accountPage.exactText(userName)).toBeVisible();
+  await expect(accountPage.exactText(userEmail)).toBeVisible();
+  await expect(accountPage.exactText(userPhoneNumber)).toBeVisible();
+  await expect(accountPage.exactText(userResidency)).toBeVisible();
+
+  // WHEN user clicks on edit button, navigate to edit page
+  await accountPage.editButton().click();
+  const editAccountPage = new UserMyAccountEditPage(context);
+  await editAccountPage.checkUrl();
```

Comment:
To properly check that edits are not persisted when the cancel button is clicked, we should make some edits in the editAccountPage before clicking cancel.

Reply:
Addressed 76415d0b7a43453a12a8e9f100ff2ea5c5349abf